TROUBLESHOOTING.md

```diff
@@ -1,20 +1,21 @@
-If you have a non-library SendGrid issue, please contact our [support team](https://support.sendgrid.com).
+If you have a non-library Twilio SendGrid issue, please contact our [support team](https://support.sendgrid.com).
 
 If you can't find a solution below, please open an [issue](https://github.com/sendgrid/sendgrid-ruby/issues).
 
 
 ## Table of Contents
 
-* [Migrating from v2 to v3](#migrating)
-* [Continue Using v2](#v2)
-* [Testing v3 /mail/send Calls Directly](#testing)
-* [Error Messages](#error)
-* [Versioning](#versioning)
-* [Environment Variables and Your SendGrid API Key](#environment)
-* [Using the Package Manager](#package-manager)
-* [Rails Specifics](#rails-specifics)
-* [Ruby Versions](#ruby-versions)
-* [Viewing the Request Body](#request-body)
+- [Table of Contents](#table-of-contents)
```

Comment:
Prefer hyphen to asterisk?

Reply:
Ah, looks like my Markdown TOC script was misconfigured.